packages/LUIS/bin/luis.js

```diff
@@ -950,7 +950,7 @@ function validateConfig(config) {
 
     assert(typeof authoringKey === 'string', `The authoringKey  ${messageTail}`);
     assert(typeof region === 'string', `The region ${messageTail}`);
-    assert(args.region == "westus" || args.region == 'westeurope' || args.region == 'australiaeast' || args.region == 'virginia', `${args.region} is not a valid authoring region.  Valid values are [westus|westeuerope|australiaest]`);
+    assert(args.region == "westus" || args.region == 'westeurope' || args.region == 'australiaeast' || args.region == 'virginia', `${args.region} is not a valid authoring region.  Valid values are [westus|westeurope|australiaeast|virginia]`);
```

Comment:
@mdrichardson, does there need to be additional logic for supporting `'virginia'`? It looks like the authoring endpoint for this region is: https://virginia.api.cognitive.microsoft.us/luis/v2.0

More [info](https://docs.microsoft.com/en-us/azure/azure-government/documentation-government-services-aiandcognitiveservices#language-understanding-luis).

Also, can you change `"westus"` to be `'westus'`? Thanks 😄

Reply:
Fixed the double quotes. As far as I know, there doesn't need to be additional logic. It was already accepted before; it just didn't show up in the "acceptable regions" message. But maybe that was intentional since it's gov?